spec/models/manageiq/providers/hawkular/datawarehouse_manager/datawarehouse_manager_spec.rb

```diff
@@ -2,6 +2,18 @@
   context "#verify_ssl_mode" do
     let(:ems) { FactoryGirl.build(:ems_hawkular_datawarehouse) }
 
+    it ".ems_type" do
+      expect(described_class.ems_type).to eq("hawkular_datawarehouse")
```

Comment:
`Datawarehouse` is almost dead. I wouldn't worry about it for coverage tests. Instead, probably we should push #24 harder to remove it from the repo.

Reply:
mmm I'll wait until #24 to continue. Thanks @israel-hdez